### ISSUE TITLE
in build.zig: `thisDependency` method fails in master

### ISSUE THREAD

**Issue by terraquad** [castholm/zigglgen#8]
version: `0.14.0-dev.1671+085cc54aa`

zig build output:
```
C:\Users\******\AppData\Local\zig\p\1220f4188a5e1bdbb15fd50e9ea322c0721384eeba9bc077e4179b0b0eeaa7fe4ad9\build.zig:54:58: error: no field named 'Struct' in union 'builtin.Type'
        const pkg_hash = inline for (@typeInfo(all_pkgs).Struct.fields) |decl| {
                                                         ^~~~~~
C:\Users\******\scoop\persist\zigup\zig\0.14.0-dev.1671+085cc54aa\files\lib\std\builtin.zig:257:18: note: union declared here
pub const Type = union(enum) {
                 ^~~~~
referenced by:
    generateBindingsSourceFile: C:\Users\******\AppData\Local\zig\p\1220f4188a5e1bdbb15fd50e9ea322c0721384eeba9bc077e4179b0b0eeaa7fe4ad9\build.zig:35:40
    generateBindingsModule: C:\Users\******\AppData\Local\zig\p\1220f4188a5e1bdbb15fd50e9ea322c0721384eeba9bc077e4179b0b0eeaa7fe4ad9\build.zig:30:55
    7 reference(s) hidden; use '-freference-trace=9' to see all references

```

**Comment by terraquad** [comment]
.

**Comment by castholm** [comment]
@terraquad Why did you close this and the two pull requests?

**Comment by castholm** [comment]
I might otherwise take a look at this later next week. I've deliberately put supporting master on hold because I want to also support the tagged 0.13.0 compiler, and with the changes to `std.builtin` multi-versioning will take a bit more effort than simply updating the code to the new field/decl names.

**Comment by terraquad** [comment]
> @terraquad Why did you close this and the two pull requests?

I thought that changing this might be a bit over the top because it removes support to the mach versions.

**Comment by terraquad** [comment]
> > @terraquad Why did you close this and the two pull requests?
> 
> I thought that changing this might be a bit over the top because it removes support to the mach versions.

I still have the changes in [here](https://github.com/terraquad/zigglgen-master)

**Comment by castholm** [comment]
I'll see if I can figure something out next week. Maybe I'll drop support for the Mach version and only support 0.13 and master if it ends up being a problem.
In the meantime, I'll reopen this.

**Comment by terraquad** [comment]
I could add compatibility layers so it uses the old `std.builtin.Type` fields when on 0.13.0 (or lower) and otherwise uses the new API.

Also the README seems out of date (The supported versions part). I could work on that too.

**Comment by castholm** [comment]
Fixed as of 9e3e04055e8271cc725774ff91292c7b877a90a4
